.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 
+before_install:
+  - if [[ "$TRAVIS_NODE_VERSION" =~ ^v6 ]]; then npm i npm -g; fi
```

Comment:
This might mean that `yargs` requires npm v6.4.1 (that's what node.js 8 currently installs).  On the other hand it might be that this requirement only applies to installs which include the development dependencies.